Library/Formula/gtk+3.rb

```diff
@@ -37,21 +30,37 @@ def install
       --disable-glibtest
       --enable-introspection=yes
       --disable-schemas-compile
+      --enable-quartz-backend
+      --enable-quartz-relocation
+      --disable-x11-backend
     ]
-
-    if build.without? "x11"
-      args << "--enable-quartz-backend" << "--enable-quartz-relocation"
-    else
-      args << "--enable-x11-backend"
-    end
+    # --enable-x11-backend not possible because of libepoxy, which is now a hard dependency of gtk3.
+    # Looks like this will remain so
+    # according to the comments in https://github.com/anholt/libepoxy/pull/28
 
     system "./configure", *args
+    # necessary to avoid gtk-update-icon-cache not being found during make install
+    mkdir_p bin
+    ENV.prepend_path "PATH", "#{bin}"
     system "make", "install"
     # Prevent a conflict between this and Gtk+2
     mv bin/"gtk-update-icon-cache", bin/"gtk3-update-icon-cache"
+    rm_f share/"glib-2.0/schemas/gschemas.compiled"
```

Comment:
this file is giving the buildbot headaches. It is generated by post_install in both gsettings-desktop-schemas and gtk+3 and is not removed when uninstalling these ports (as it should).